mappings/services.json

```diff
@@ -1,4 +1,27 @@
 {
+  "settings": {
+    "index": {
+      "number_of_shards": 1
+    },
+    "analysis": {
+      "analyzer": {
+        "stemming_analyzer": {
+          "tokenizer": "standard",
+          "filter": ["standard", "lowercase", "possessive_english_stemmer", "light_english_stemmer"]
+        }
+      },
+      "filter": {
+        "light_english_stemmer": {
+          "type": "stemmer",
+          "name": "light_english"
+        },
+        "possessive_english_stemmer": {
```

Comment:
@Wynndow Sam left a comment about it here: https://trello.com/c/TPWu8MX4/337-understand-decide-the-order-of-search-results#comment-59789a487c7999c0d056a448